app/javascript/components/analytics/widgets/MonitoreesByEventDate.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { Card, Form, Col, Row } from 'react-bootstrap';
-import { ResponsiveContainer, BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip } from 'recharts';
+import { Card, Col, Form, Row } from 'react-bootstrap';
+import { Bar, BarChart, CartesianGrid, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
```

Comment:
Only added `import { mapToChartFormat, parseOutFields } from '../../../lib/AnalyticsUtils';`. Everything else here is just moving `imports` around / alphabetizing them.